services/bookables-api/helpers/bookables.js

```diff
@@ -0,0 +1,21 @@
+import { throwError } from '@helsingborg-stad/npm-api-error-handling';
+import to from 'await-to-js';
+
+import params from '../../../libs/params';
+import config from '../../../config';
+
+async function getBookables() {
+  const { bookables } = await getSsmParameters();
+  return bookables;
+}
+
+async function getSsmParameters() {
```

Comment:
I am having an idea that we can remove this function `getSsmParamters` and instead use the `throwError` function directly in the params.read function. I think that would be better instead of duplicate this function all the time.

Reply:
Yes, we should discuss this with the rest of the team and do a separate pull request for it where we remove the function from all services. 
For now we'll keep the function for the sake of consistency with the other services.